compiler/test/stdlib/pervasives.test.gr

```diff
@@ -0,0 +1,54 @@
+// compare
+
+// Number comparisons are tested thoroughly by other tests
+assert compare(0, 1) < 0
+assert compare(0, 0) == 0
+assert compare(1, 0) > 0
```

Comment:
I think we should have a test for `NaN` (given the code comments throughout) and maybe even `Infinity` (?)

Reply:
I could have sworn I added some!